roles/splunk_universal_forwarder/tasks/main.yml

```diff
@@ -1,50 +1,32 @@
 ---
-- include_tasks: ../../splunk_common/tasks/install_splunk.yml
-
-- name: Generate user-seed.conf
-  ini_file:
-    dest: "{{ splunk.home }}/etc/system/local/user-seed.conf"
-    section: user_info
-    option: "{{ item.opt }}"
-    value: "{{ item.val }}"
-  with_items:
-    - { opt: 'USERNAME', val: 'admin' }
-    - { opt: 'PASSWORD', val: '{{ splunk.password }}' }
-  loop_control:
-    label: "{{ item.opt }}"
-  no_log: "{{ hide_password }}"
-
-- include_tasks: pre_splunk_start_commands.yml
```

Comment:
Is this play still required for forwarders? Looks like they are not executing those tasks anymore.

Reply:
I moved this to the common role, so technically all roles can utilize the `SPLUNK_BEFORE_START_CMD` env to run something before Splunk starts